qrenderdoc/Windows/MainWindow.cpp

```diff
@@ -2178,7 +2178,10 @@ void MainWindow::OnCaptureLoaded()
 
   statusProgress->setVisible(false);
 
-  ui->action_Recompress_Capture->setEnabled(true);
+  // don't allow capture recompress on opened images
+  QString driver = m_Ctx.Replay().GetCaptureAccess()->DriverName();
+  bool is_image = driver == lit("Image");
+  ui->action_Recompress_Capture->setEnabled(!is_image);
```

Comment:
The purist in me wants to expose this somehow via an enum properly (especially since there is one internally), but realistically it's too much plumbing to be worthwhile.

Reply:
You would better know how to do it :) Thinking of it now... maybe "Save" and "Save As" should also be disabled for images?

Reply:
Yes probably they should too!